lib/discourse_ip_info.rb

```diff
@@ -2,6 +2,8 @@
 
 require 'maxminddb'
 require 'resolv'
+require 'rubygems/package'
```

Comment:
I think these requires are too high, just require them in the methods that use them cause the vast majority of time we don't need to pull in either.